packages/graphql/src/classes/Neo4jGraphQL.ts

```diff
@@ -19,8 +19,7 @@
 
 import type { Driver } from "neo4j-driver";
 import type { DocumentNode, GraphQLSchema } from "graphql";
-import type { IExecutableSchemaDefinition } from "@graphql-tools/schema";
-import { makeExecutableSchema } from "@graphql-tools/schema";
+import { addResolversToSchema, IExecutableSchemaDefinition, makeExecutableSchema } from "@graphql-tools/schema";
```

Comment:
Very particular nitpick here, but I think it would be nicer to keep the `type` imports separated from the normal imports.

Reply:
Errrrrr, TypeScript should be enforcing this for us unless someone has changed the config? 🧐

Reply:
It seems that when they are mixed, TypeScript doesn't look after this for us - want to raise that proposal as a separate PR?

Reply:
Will do!